crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -583,53 +665,91 @@ pub fn get_bind_group_layout_entries(
             },
             count: None,
         },
+        // Deferred texture
+        BindGroupLayoutEntry {
+            binding: bindings[3],
+            visibility: ShaderStages::FRAGMENT,
+            ty: BindingType::Texture {
+                multisampled: false,
+                sample_type: TextureSampleType::Uint,
+                view_dimension: TextureViewDimension::D2,
+            },
+            count: None,
+        },
     ]
 }
 
-pub fn get_bindings<'a>(
-    prepass_textures: Option<&'a ViewPrepassTextures>,
-    fallback_images: &'a mut FallbackImagesMsaa,
-    fallback_depths: &'a mut FallbackImagesDepth,
-    msaa: &'a Msaa,
-    bindings: [u32; 3],
-) -> [BindGroupEntry<'a>; 3] {
+// Needed so the texture views can live long enough.
```

Comment:
why do they need to live longer now than before?

Reply:
This has to do with not being able to mutably borrow `fallback_images` more than once at a time since the output of the function keeps a reference to both instances.